content/build/cloud/ci.md

```diff
@@ -0,0 +1,366 @@
+registry directly, rather than loading the image and then pushing it. Pushing
+directly speeds up your builds and avoids unnecessary file transfers.
+
+If you just want to build and discard the output, export the results to the
+build cache or build without tagging the image. When you use Docker Cloud Build,
+Buildx automatically loads the build result if you build a tagged image.
+See [Loading build results](./usage/#loading-build-results) for details.
+
+> **Note**
+>
+> Builds on Docker Cloud Build have a timeout limit of two hours. Builds that
+> run for longer than two hours are automatically cancelled.
+
+{{< tabs >}}
+{{< tab name="GitHub Actions" >}}
+
+```yaml
+name: ci
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PAT }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          version: "lab:latest"
+          driver: cloud
+          endpoint: "<ORG>/default"
+          install: true
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          tags: "<IMAGE>"
+          # For pull requests, export results to the build cache.
+          # Otherwise, push to a registry.
+          outputs: ${{ github.event_name == 'pull_request' && 'type=cacheonly' || 'type=registry,push=true' }}
+```
+
+{{< /tab >}}
+{{< tab name="GitLab" >}}
+
+```yaml
+default:
+  image: docker:24-dind
+  services:
+    - docker:24-dind
+  before_script:
+    - docker info
+    - echo "$DOCKER_PAT" | docker login --username "$DOCKER_USER" --password-stdin
+    - |
+      apk add curl jq
+      ARCH=${CI_RUNNER_EXECUTABLE_ARCH#*/}
+      BUILDX_URL=$(curl -s https://raw.githubusercontent.com/docker/actions-toolkit/main/.github/buildx-lab-releases.json | jq -r ".latest.assets[] | select(endswith(\"linux-$ARCH\"))")
+      mkdir -vp ~/.docker/cli-plugins/
+      curl --silent -L --output ~/.docker/cli-plugins/docker-buildx $BUILDX_URL
+      chmod a+x ~/.docker/cli-plugins/docker-buildx
+    - docker buildx create --use --driver cloud ${DOCKER_ORG}/default
+
+variables:
+  IMAGE_NAME: <IMAGE>
+  DOCKER_ORG: <ORG>
+
+# Build multi-platform image and push to a registry
+build_push:
+  stage: build
+  script:
+    - |
+      docker buildx build \
+        --platform linux/amd64,linux/arm64 \
+        --tag "${IMAGE_NAME}:${CI_COMMIT_SHORT_SHA}" \
+        --push .
+
+# Build an image and discard the result
+build_cache:
+  stage: build
+  script:
+    - |
+      docker buildx build \
+        --platform linux/amd64,linux/arm64 \
+        --tag "${IMAGE_NAME}:${CI_COMMIT_SHORT_SHA}" \
+        --output type=cacheonly \
+        --push .
```

Comment:
typo: you can't do `--output type=cacheonly` AND `--push`, and regarding the section pretty sure `--push` is not what you want :)